src/sql-parser/src/ast/defs/query.rs

```diff
@@ -362,10 +362,12 @@ impl<T: AstInfo> AstDisplay for CteBlock<T> {
                     f.write_str("WITH ");
                     f.write_node(&display::comma_separated(list));
                 }
-                CteBlock::MutuallyRecursive(MutRecBlock { max_depth, ctes }) => {
+                CteBlock::MutuallyRecursive(MutRecBlock { max_iterations, ctes }) => {
                     f.write_str("WITH MUTUALLY RECURSIVE ");
-                    if let Some(max_depth) = max_depth {
-                        f.write_node(max_depth);
+                    if let Some(max_iterations) = max_iterations {
```

Comment:
This needs a test in https://github.com/MaterializeInc/materialize/blob/main/src/sql-parser/tests/testdata/select

Reply:
Thanks! Just saw your msg on Slack. I'll definitely add a test.

(I'll ping you when the PR is ready for review.)

Reply:
Done.